nix-support/mypy.nix

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
rather than writing a new definition, can you just use overrideAttrs with a new `src` and `version` attr?

Reply:
I guess it might be tricky due to the way the python packaging infra works?

Reply:
I wrote it that way expecting it to be awkward to update given all that's going on in the original, but it turned out to be easier than expected.

I've changed it to an overrideAttrs approach now. The annoying part is that I need to override the name manually.